.gitignore

```diff
@@ -14,3 +14,8 @@ vendor
 old/
 
 .remarkrc
+
+tar_staging.sh
+package.tgz
+
+design-system
```

Comment:
what creates this dir?

Reply:
A deploy bash script that will facilitate updating staging when Travis is down. I didn't want to upload it to the public repo; I was going to drop it into a private ops playbook repo.